xlators/cluster/dht/src/dht-rebalance.c

```diff
@@ -4110,9 +4100,6 @@ gf_defrag_parallel_migration_init(xlator_t *this, gf_defrag_info_t *defrag,
     int index = 0;
     pthread_t *tid = NULL;
 
-    if (!defrag)
```

Comment:
Similar check is there in gf_defrag_parallel_migration_cleanup(), which seems redundant as well.